### ISSUE TITLE
Package build has not updated after PR merge

### ISSUE THREAD

**Issue by neoformit** [bioconda/bioconda-recipes#32263]
[This PR for smudgeplot](https://github.com/bioconda/bioconda-recipes/pull/32056) was merged 12 days ago, but when I `conda install` the package it still gives me the old build (no. 1). The PR was for build no.2.

`conda search -c bioconda -i smudgeplot` shows this as the latest build (25th March, 9 months ago):

```sh
smudgeplot 0.2.4 py39r40h779adbc_1
----------------------------------
file name   : smudgeplot-0.2.4-py39r40h779adbc_1.tar.bz2
name        : smudgeplot
version     : 0.2.4
build       : py39r40h779adbc_1
build number: 1
size        : 92 KB
license     : Apache-2.0
subdir      : linux-64
url         : https://conda.anaconda.org/bioconda/linux-64/smudgeplot-0.2.4-py39r40h779adbc_1.tar.bz2
md5         : 65a9b06324250b26fc0ee8bf942d68e1
timestamp   : 2021-03-25 02:39:57 UTC
dependencies:
  - libgcc-ng >=9.3.0
  - numpy
  - python >=3.9,<3.10.0a0
  - r-argparse
  - r-base >=4.0,<4.1.0a0
  - r-viridis
  - scipy
```

Any idea why the new build is not available?

**Comment by bgruening** [comment]
I guess your base python version is smaller than 3.10 and you only have build-2 packages available for python 3.10.

If you look at https://anaconda.org/bioconda/smudgeplot/files you see that the `_2` builds are only available for python 3.10. So if you install your package with python 3.9 you will get `_1` as this is the only available version for that python version.

**Comment by neoformit** [comment]
Thanks Bjorn I think that takes us back to [my last issue](https://github.com/bioconda/bioconda-recipes/issues/32088), I'll close this.
